python/samples/getting_started/workflows/agents/azure_ai_agents_streaming.py

```diff
@@ -66,8 +67,14 @@ async def main() -> None:
                 "Provide the feedback in the most concise manner possible."
             ),
         )
-
-        workflow = WorkflowBuilder().set_start_executor(writer).add_edge(writer, reviewer).build()
+        # Add agents to workflow with custom settings using add_agent.
+        # Agents adapt to workflow mode: run_stream() for incremental updates, run() for complete responses.
+        # Reviewer agent emits final AgentRunResponse as a workflow output.
+        builder = WorkflowBuilder()
```

Comment:
Not chained?

Reply:
It works without chaining if that's what you are asking. Updated to use chaining as in the future we may want to make the builder to support concurrency.